src/platforms/ios/iosiaphandler.mm

```diff
@@ -24,6 +25,7 @@
 
 namespace {
 Logger logger(LOG_IAP, "IOSIAPHandler");
+bool transactionsProcessed = false;
```

Comment:
Why is this in the namespace and not some kind of m_transactionsProcessed? (This is a real question and not me asking you to change it - I am not good experienced enough at cpp to have strong opinions about this out the gate - just curious).

Reply:
Since we call it in `IOSIAPHandlerDelegate` we cannot access member variables of `IOSIAPHandler`. There are ways to do that but using namespace was just more straightforward.

Reply:
Rename it s_transactionsProcessed